src/utils/exec.ts

```diff
@@ -23,28 +46,11 @@ export async function exec(dir: string, cliArgs: string): Promise<string> {
       },
       (error, result) => {
         if (error) {
-          reject(error);
+          return reject(error);
         }
 
-        resolve(typeof result === 'string' ? result : `${result}`);
+        resolve(String(result));
       },
     );
   });
```

Comment:
suggestion: if we're generally refactoring for clarity in this PR, it seems like you could just `promisify` the exec call.